plugins/modules/eos_interfaces.py

```diff
@@ -83,12 +83,26 @@
         description:
         - Interface link speed. Applicable for Ethernet interfaces only.
         type: str
+  running_config:
+    description:
+      - This option is used only with state I(parsed).
+      - The value of this option should be the output received from the NX-OS device by executing
+        the command B(show running-config | section ^interface).
+      - The state I(parsed) reads the configuration from C(running_config) option and transforms
+        it into Ansible structured data as per the resource module's argspec and the value is then
+        returned in the I(parsed) key within the result.
+    type: str
+    version_added: "1.0.0"
```

Comment:
Should this not be on the module instead of the option? We want `version_added: 1.0.0` on all the modules to avoid confusion about when they were added, but we don't need it on all the options if they were added at the same time.